plugins/app/src/apis/SwappableComponentsApi/DefaultSwappableComponentsApi.tsx

```diff
@@ -71,4 +71,14 @@ export class DefaultSwappableComponentsApi implements SwappableComponentsApi {
       return <DefaultComponent {...innerProps} />;
     };
   }
+
+  componentIsLoadable(ref: SwappableComponentRef<any>): boolean {
+    if (!this.#components.has(ref.id)) {
+      return false;
+    }
+    const OverrideComponent = this.#components.get(ref.id);
+    const { defaultComponent: DefaultComponent } =
+      OpaqueSwappableComponentRef.toInternal(ref);
+    return Boolean(OverrideComponent || DefaultComponent);
```

Comment:
Just noticed this might not work because there is the `FallbackComponent` defined, @Rugvip any pointers how to make this https://github.com/backstage/backstage/pull/31344/files#diff-5fb0067db499739a8def2b948fb25eed16fcfa5c430a77d18221bd2da6adc8b8R94 happen?